pkgs/build-support/rust/build-rust-crate.nix

```diff
@@ -19,7 +19,7 @@ let makeDeps = dependencies:
       echo_build_heading() {
        start=""
        end=""
-       if [[ x"${colors}" -eq x"always" ]]; then
+       if [[ x"${colors}" = x"always" ]]; then
```

Comment:
Is this `x"` stuff needed in any posix shell from this century?

Reply:
I have absolutely no idea - I made the minimal changes to make it work. I'm not very good at bash.